src/main/java/org/opensearch/sdk/ScriptExtension.java

```diff
@@ -0,0 +1,37 @@
+package org.opensearch.sdk;
```

Comment:
Needs a license header. Running `./gradlew spotlessApply` should add it for you.